mirgecom/logging_quantities.py

```diff
@@ -428,6 +428,13 @@ def __call__(self) -> Optional[float]:
             warn(f"cudaMemGetInfo failed with error {ret}.")
             return None
         else:
+            if self.free.value / self.total.value < 0.1:
```

Comment:
While I'm at it, why are `self.free` and `self.total` instance attributes? I don't see a need for them to be long-lived.

Reply:
That's a good catch, thanks! Unfortunately, I can't remember why I wrote the code this way in #255 (possibly to reduce overhead from the `ctypes.c_size_t()` calls?). I can't see a performance difference with [844db39](https://github.com/illinois-ceesd/mirgecom/pull/874/commits/844db3950e53f4a04fdd24b6363b2d447de2d348), so it should be ok to change these to local variables.